src/js/modules/sections/components/SectionPage.js

```diff
@@ -36,33 +131,48 @@ const SectionPage = ({ classes, articles, subsections, section, match }) => {
         pathToArticlePage = article.sectionSlug + '/' + article.slug;
       }
       return (
-        <li key={article.id}>
-          <Link to={match.url + '/' + pathToArticlePage}>{article.title}</Link>
-        </li>
+        <div>
+        <Row key={article.id} className={classes.eachArticle}>
+          <Col md={3} lg={3}>
+            <figure>
+              <img src={featuredMedia.url} className={classes.image}/>
+            </figure>
+          </Col>
+          <Col md={6} lg={6}>
+            <Link to={match.url + '/' + pathToArticlePage} className={classes.articleTitle}>{article.title}</Link>
+            <p className={classes.articleDescription}>Cathy plays a horizontal guitara.</p>
+            <p className={classes.articleAuthor}>Jason Kao</p>
+          </Col>
+        </Row>
+        <hr className={classes.articleDivider}/>
+        </div>
       );
     });
   };
   return (
     <div className={classes.SectionPage}>
-      <h1>{section.name}</h1>
-      <p>description: {section.description}</p>
-      <hr/>
-      <p>subsections</p>
-      <ul>
+      <h1 className={classes.sectionName}>{section.name}</h1>
+      {/* <p>description: {section.description}</p> */}
+      <ul className={classes.subsectionLink}>
         {createSubsectionLinks()}
       </ul>
-      <hr/>
-      <p>articles</p>
-      <ul>
+      <hr className={classes.sectionNameDivider}/>
+      <Grid className={classes.articleList}>
```

Comment:
Make the entire articleList its own component: call it `<ArticleList/>`. Make sure to pass `articles` into the `<ArticleList/>` as props. You have `articles` already because `articles` is a prop for the `SectionPage` (see line 169, then line 114). Alsp pass in `featuredMedia` as props.